docs/services/vision/detection.md

```diff
@@ -0,0 +1,319 @@
+2. Enter a name as the **Name**.
+3. Click **Create Service**.
+
+<img src="../../../tutorials/img/try-viam-color-detection/create-service.png" alt="The Create Service panel lists the type as vision and name as vision, with a Create Service button.">
+
+In your Vision Service's panel, add a detector into the **Attributes** field.
+For example:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+ "register_models": [
+   {
+     "name": "my_color_detector",
+     "type": "color_detector",
+     "parameters": {
+       "detect_color": "#7a4f5c",
+       "hue_tolerance_pct": 0.06,
+       "segment_size_px": 100
+     }
+   }
+ ]
+}
+```
+
+Click **SAVE CONFIG** and head to the **COMPONENTS** tab.
+
+{{%expand "You can also configure the entire Vision Service and detector in raw JSON" %}}
+
+To add a vision model to your robot, add the `name`, `type`, and `parameters` of the desired detector to the `register_models` in the attributes field of the Vision Service config.
+For example:
+
+``` json {class="line-numbers linkable-line-numbers"}
+"services": [
+    {
+        "name": "vision1",
+        "type": "vision",
+        "attributes": {
+          "register_models": [
+            {
+              "name": "my_color_detector",
+              "type": "color_detector",
+              "parameters": {
+                "detect_color" : "#A3E2FF",
+                "hue_tolerance_pct": 0.06,
+                "segment_size_px": 100
+              }
+            },
+            {
+              "name": "my_classifier",
+              "type": "tflite_classifier",
+              "parameters": {
+                "model_path" : "/path/to/model.tflite",
+                "label_path": "/path/to/labels.txt",
+                "num_threads": 1
+              }
+            },
+            {
+                "name": "my_segmenter",
+                "type": "radius_clustering_segmenter",
+                "parameters": {
+                    "min_points_in_plane": 2,
+                    "min_points_in_segment": 2,
+                    "clustering_radius_mm": 3.0,
+                    "mean_k_filtering": 0
+                }
+            }
+          ]
+        }
+    }
+]
+```
+
+{{% /expand%}}
+
+### Add a camera component and a "transform" model
+
+You cannot interact directly with the [Vision Service](/services/vision/).
+To be able to interact with the Vision Service you must:
+
+1. Configure a physical [camera component](../../../components/camera) to wrap the service.
+2. Configure a [transform camera](../../../components/camera/transform) to view output from the detector overlaid on images from the physical camera.
+
+After adding the component and its attributes, click **SAVE CONFIG**.
+
+Wait for the robot to reload, and then go to the **CONTROL** tab to test the stream of detections.
+
+## Code
+
+The following code gets the robot’s Vision Service and then runs a color detector vision model on an image from the robot's camera `"camera_1"`:
```

Comment:
[probably out of PR scope] This usage of "Vision Service" makes me wonder whether we should capitalize services in the context of "the Viam Vision Service allows you to do fancy things" but not in the context of a given instance of the service

Reply:
Let's discuss it in our next team meeting, I think it's a good idea - but I don't want to hold up the PR for it. I will make the change here for now. 
```suggestion
The following code gets the robot’s vision service and then runs a color detector vision model on an image from the robot's camera `"camera_1"`:
```